socorro/external/postgresql/adi.py

```diff
@@ -51,19 +51,40 @@ def get(self, **kwargs):
                 product_adu
             LEFT OUTER JOIN product_versions pv USING (product_version_id)
             WHERE
-                pv.product_name = %(product)s
-                AND pv.version_string IN %(versions)s
-                AND os_name IN %(platforms)s
-                AND adu_date BETWEEN %(start_date)s AND %(end_date)s
+                pv.product_name = %%s
+                AND (%s)
+                AND os_name IN %%s
+                AND adu_date BETWEEN %%s AND %%s
```

Comment:
Ah my bad, I didn't see that you were doing a tuple formatting for the versions. Well it's fine then. It's still possible to make it a keyword formatting, but it will make the logic ugly, so let's forget it.